src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100ServiceTest.java

```diff
@@ -723,15 +831,288 @@ void testC100RespondentTweNotLivedInAddressForFiveYearsWarning() {
+                                WITHOUTNOTICE_JURISDICTIONISSUE_DETAILS.equalsIgnoreCase(
+                                        eachField.getName()))
+                .forEach(field -> field.setValue(EMPTY_STRING));
+
+        BulkScanValidationResponse res =
+                bulkScanValidationService.validate(bulkScanValidationRequest);
+
+        assertEquals(Status.WARNINGS, res.status);
+        assertTrue(res.getWarnings().items.contains(INTERNATIONAL_JURISDICTION_WARNING_MESSAGE));
+    }
+
+    @Test
+    @DisplayName(
+            "Should generate warning for checkbox ticked with 'Yes' when a requesthas been made or"
+                    + "should a request be made to a Central Authority or other competent "
+                    + "authority in a foreign state or a consular authority in England and Wales"
+                    + "on sections 8 of application form but no details were given.")
+    void testC100InternationalElementRequestCentralConsularWithoutDetailsWarning() {
+        List<OcrDataField> c100GetRequestCentralConsularWarningData = new ArrayList<>();
+        c100GetRequestCentralConsularWarningData.addAll(TestDataC100Util.getData());
+
+        when(postcodeLookupService.isValidPostCode(POST_CODE, null)).thenReturn(true);
+        BulkScanValidationRequest bulkScanValidationRequest =
+                BulkScanValidationRequest.builder()
+                        .ocrdatafields(c100GetRequestCentralConsularWarningData)
+                        .build();
+
+        bulkScanValidationRequest.getOcrdatafields().stream()
+                .filter(
+                        eachField ->
+                                INTERNATIONAL_OR_FACTORS_AFFECTING_LITIGATION_FIELD
+                                        .equalsIgnoreCase(eachField.getName()))
+                .forEach(field -> field.setValue(YES));
+
+        bulkScanValidationRequest.getOcrdatafields().stream()
+                .filter(
+                        eachField ->
+                                INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH.equalsIgnoreCase(
+                                        eachField.getName()))
+                .forEach(field -> field.setValue("Yes"));
+
+        bulkScanValidationRequest.getOcrdatafields().stream()
+                .filter(
+                        eachField ->
+                                INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH_DETAILS
+                                        .equalsIgnoreCase(eachField.getName()))
+                .forEach(field -> field.setValue(EMPTY_STRING));
+
+        BulkScanValidationResponse res =
+                bulkScanValidationService.validate(bulkScanValidationRequest);
+
+        assertEquals(Status.WARNINGS, res.status);
+        assertTrue(
+                res.getWarnings()
+                        .items
+                        .contains(
+                                INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH_WARNING_MESSAGE));
+    }
+
```

Comment:
can you move to different C100 test class pls, as it is to big.???

Reply:
Moved to BulkScanC100ValidationServiceTest class